app/src/main/java/com/etesync/syncadapter/ui/ChangeEncryptionPasswordActivity.kt

```diff
@@ -72,7 +72,7 @@ open class ChangeEncryptionPasswordActivity : BaseActivity() {
             val httpClient = HttpClient.Builder(this@ChangeEncryptionPasswordActivity).setForeground(true).build().okHttpClient
 
             try {
-                Logger.log.info("Loging in with old password")
+                Logger.log.info("Logging in with old password")
```

Comment:
It's an internal log, let's keep it the way it is so it's easier to search for it.